classes/controller/AdminController.php

```diff
@@ -436,38 +436,48 @@ public function __construct($forceControllerName = '', $default_theme_name = 'de
         $token = $this->token;
 
         $this->_conf = array(
-            1 => $this->l('Successful deletion'),
-            2 => $this->l('The selection has been successfully deleted.'),
-            3 => $this->l('Successful creation'),
-            4 => $this->l('Successful update'),
-            5 => $this->l('The status has been successfully updated.'),
-            6 => $this->l('The settings have been successfully updated.'),
-            7 => $this->l('The image was successfully deleted.'),
-            8 => $this->l('The module was successfully downloaded.'),
-            9 => $this->l('The thumbnails were successfully regenerated.'),
-            10 => $this->l('The message was successfully sent to the customer.'),
-            11 => $this->l('Comment successfully added'),
-            12 => $this->l('Module(s) installed successfully.'),
-            13 => $this->l('Module(s) uninstalled successfully.'),
-            14 => $this->l('The translation was successfully copied.'),
-            15 => $this->l('The translations have been successfully added.'),
-            16 => $this->l('The module transplanted successfully to the hook.'),
-            17 => $this->l('The module was successfully removed from the hook.'),
-            18 => $this->l('Successful upload'),
-            19 => $this->l('Duplication was completed successfully.'),
-            20 => $this->l('The translation was added successfully, but the language has not been created.'),
-            21 => $this->l('Module reset successfully.'),
-            22 => $this->l('Module deleted successfully.'),
-            23 => $this->l('Localization pack imported successfully.'),
-            24 => $this->l('Localization pack imported successfully.'),
-            25 => $this->l('The selected images have successfully been moved.'),
-            26 => $this->l('Your cover image selection has been saved.'),
-            27 => $this->l('The image\'s shop association has been modified.'),
-            28 => $this->l('A zone has been assigned to the selection successfully.'),
-            29 => $this->l('Successful upgrade'),
-            30 => $this->l('A partial refund was successfully created.'),
-            31 => $this->l('The discount was successfully generated.'),
-            32 => $this->l('Successfully signed in to PrestaShop Addons')
+            1 => $this->trans('Successful deletion', array(), 'Admin.Notifications.Success'),
```

Comment:
Some strings end with a full-stop, some don't. Could you make them all end with a "." for consistency? Thank you!